scripts/module/spectate.nut

```diff
@@ -1,8 +1,8 @@
 function onKeyDown( player, key ) {
 
-  if(player.Spawned) {
-    return;
-  }
+//  if(player.Spawned) {
+//    return;
+//  }
```

Comment:
Leaving dead code is bad.

We should have some `can_always_spectate <- true;` (near the top) which disables this check (and is self-describing).
Alternatively, if we decide we never want to restrict spectating this way: remove this code altogether.

But commenting out unused code is bad practice.